core/src/main/scala/caliban/GraphQL.scala

```diff
@@ -105,16 +105,18 @@ trait GraphQL[-R] { self =>
                                                           }
                   typeToValidate                        = if (intro) introspectionRootType else rootType
                   schemaToExecute                       = if (intro) introspectionRootSchema else schema
-                  validatedReq                         <- VariablesCoercer.coerceVariables(request, doc, typeToValidate, skipValidation)
+                  validatedVariables                   <-
+                    VariablesCoercer
```

Comment:
This is super minor but how about extracting `request.variables.getOrElse(Map.empty)` into a variable so that hopefully this fits in one line and stays easier to read?

Reply:
I was a bit wary of extracting them into a variable so we don't accidentally use them elsewhere but yeah the formatting was annoying me as well. Hopefully naming them `unsafeVars` (`unsafeVariables` didn't fit in a single line) will be clear enough that they're not to be used 😅